Test/JsonDiffTest.cs

```diff
@@ -66,22 +66,28 @@ public static IEnumerable<object[]> ProviderForNumericDiffTest()
                                      { 'b': 10 },
                                      { 'c': 20 },
                                     ]
-                                 }", 
+                                 }",
                                @"{
                                'a': {
                                     '_t': 'a',
                                      '0': {
                                            'b': {
                                                    'left': 15,
                                                    'right': 10,
-                                                    'diff': 5
+                                                    'Int Diff': {
+                                                      'Subtraction': 5,
```

Comment:
The usual convention is that subtraction is right minus left, not left minus right. I would change this